benchmarks/templates/benchmarks/leaderboard/leaderboard-table.html

```diff
@@ -0,0 +1,151 @@
+{% if models %}
+{% if has_user %}
+
+
+<form>
+{% endif %}
+
+<banner>
+  <div class="leaderboard-table-banner-icon gradient">
+      <img src="{% static "/benchmarks/img/leaderboard_header_icon.png" %}" alt="Brain-Score Logo">
+  </div>
+  <div class="leaderboard-table-banner box gradient"><h3>Leaderboard</h3></div>
+</banner>
+<div class="filters">
+  <select class="benchmark-sort">
+    <option value="" disabled selected>Sort by Benchmarks</option>
+    <option value="average">Sort by average</option>
+    {% for name in benchmark_names %}
+      <option value="{{name}}">{{ name }}</option>
+    {% endfor %}
+  </select>
+  <div class="legend">
+    <label>Score Legend</label>
+    <span>No Alignment</span>
+    <span class="legend-right">Alignment at ceiling</span>
+  </div>
+  <div class="leaderboard-breadcrumb"></div>
+</div>
+<div class="leaderboard-table-container">
+  <table class="table">
+    <thead>
+      <tr>
+        <th class="rank-header">
+          <p>Rank</p>
+        </th>
+        <th class="model-header">
+          <p>Model</p>
+        </th>
+        <!
+          Each of the cells in table/heading have a lot of information
+          of what benchmark they are and who their parent is. This allows
+          for QuerySelecting and using some of the filters from index.py
+        >
+        {% for benchmark in benchmarks %}
+          <th data-benchmark="{{ benchmark.short_name }}"
+              data-parent="{{ benchmark_parents|get_parent_item:benchmark.identifier }}"
+              data-depth="{{benchmark.depth}}"
+              class="{% if 'engineering' in benchmark.root_parent %}
+                  engineering
+                {% elif 'average' in benchmark.root_parent %}
+                  average
+                {% endif %}
+                {% if benchmark.identifier in uniform_parents and benchmark.short_name != 'average_{{ domain }}' %}
+                  cursor--pointer
+                {% endif %}
+                benchmark-header depth_{{ benchmark.depth }}"
+              style="display:{{ not_shown_set|in_set:benchmark.identifier }}"
+              title="{{ benchmark.short_name }} {% if benchmark.ceiling and benchmark.ceiling != 'X' %}ceiling: {{ benchmark.ceiling }} {% endif %}"
+          >
+            <div>
+              <p>
+                {# reference link #}
+                {% if benchmark.benchmark_type.reference and benchmark.benchmark_type.reference.url %}
+                  <a href="{{ benchmark.benchmark_type.reference.url }}" target="_blank" rel="noopener noreferrer">
+                {% endif %}
+                {# identifier #}
+                <span class="depth_{{ benchmark.depth }}
+                           {% if benchmark.version %} instance {% endif %}
+                           {% if benchmark.root_parent == 'average_{{ domain }}' %} brain_benchmark {% endif %}"
+                      value="{{ benchmark.identifier }}">{{ benchmark.short_name|simplify_domain }}
```

Comment:
Two things: 

1. Can we make the "roots" (what is displayed by default) of vision: `average`, `neural`, `behavioral` and `engineering`, just like language already does? I believe the db already has this info available.
2. For language, can we change the root names from `neural_language` and `behavioral_language` to just neural and `behavioral`?

Reply:
re 1: the info is not in the db, but at least for language the leaderboard "self-assembles" in the right form with those changes